components/dashboard/src/Menu.tsx

```diff
@@ -489,6 +493,18 @@ export default function Menu() {
                         ))}
                     </nav>
                 )}
+                {isLP && (
```

Comment:
**issue**: Not sure where can we move this piece of code, but the alert component component should be placed below the top navigation bar, including the tabs and above the pager heading. The current placement is breaking the layout in some pages.

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-07-18 at 6 13 55 PM (2)" src="https://user-images.githubusercontent.com/120486/179553078-74503267-2fbd-4346-9706-c32f1ce4650b.png"> | <img width="1440" alt="Screenshot 2022-07-18 at 6 17 16 PM (2)" src="https://user-images.githubusercontent.com/120486/179553097-a1b3365c-e53c-40ef-9b71-d42efd078e80.png"> |
| <img width="1440" alt="Screenshot 2022-07-18 at 6 57 40 PM (2)" src="https://user-images.githubusercontent.com/120486/179552993-3e6b2bcf-56d6-47fd-ab03-27e993bcf569.png"> | <img width="1440" alt="Screenshot 2022-07-18 at 6 57 32 PM (2)" src="https://user-images.githubusercontent.com/120486/179553012-b591e2c3-72b8-4824-acf9-8ef4ad63f06f.png"> |

Reply:
@gtsiolis Can you take a look on this layout issue?

I've moved this into `App.tsx` where the new AppNotifications are, and you can access the instance at https://tar-lp-out-nudge.preview.gitpod-dev.com/workspaces

Reply:
Looks great, @Pothulapati! Two minor issues:

1. **Grid**: We're using a **page container** with a **fixed layout** for our **grid layout**. Using this approach the ideal maximum width of the page container should follow the fixed width of the page container.
2. **Spacing**: We've been using an **8-point grid system** with **spacing scale** to help us provide all the negative area needed between elements and components, see [relevant discussion](https://gitpod.slack.com/archives/C01KPEPQYE6/p1657294999865219) (internal). Every part of the user interface should be intentional about empty space between elements. The amount of space between items creates relationships and hierarchy. For example, `8px` is used to separate related elements while `16px` is used to separate unrelated elements.

Cc @gitpod-io/engineering-self-hosted for visibility and future reference.

One interesting point we may run into the future could be to see how this alert stacks with more global alerts coming through the `AppNotifications.txt`, which could be fine as long as we limit the amount of alerts we stack, but I'd assume there's little to zero changes of running into more alerts here as the local preview is not intended for production use.

To get back to the actual UX changes, using a parent `div` container for the alert with two classes (`app-container` and `mt-2`) should make all the necessary changes here. 🍵 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-08-01 at 3 56 30 PM" src="https://user-images.githubusercontent.com/120486/182155887-e2466a9e-6986-44e0-8d60-2edd8386f8c9.png"> | <img width="1440" alt="Screenshot 2022-08-01 at 3 56 25 PM" src="https://user-images.githubusercontent.com/120486/182155893-b221f21c-0aa2-4a66-b4c3-73f0013344c5.png"> |

Reply:
> I'd assume there's little to zero changes of running into more alerts here as the local preview is not intended for production use.

Exactly, and hence my reason for not using `AppNotifications`.